pyout.py

```diff
@@ -674,8 +716,62 @@ def _maybe_write_header(self):
             ## We're at the header, so there aren't any previous
             ## lines to update.
             pass
-        self._writerow(row, style=self._style["header_"], adopt=False,
-                       transform=transform)
+        self._writerow(row, style=self._style["header_"], adopt=False)
+
+    def _strip_callables(self, row):
```

Comment:
this method seems to be static indeed, will include this declaration/change in my commit.  so we saw/new that